content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx

```diff
@@ -33,23 +37,47 @@ Make sure your Prisma schema is in sync with your database schema. This should a
    prisma db pull
    ```
 
-### Initialize migration history
+### Create a baseline migration
+
+Baselining is the process of initializing a migration history for a database that:
+
+- ✔ Existed before you started using Prisma Migrate
+- ✔ Contains data that must be maintained (like production), which means that the database cannot be reset
+
+Baselining tells Prisma Migrate to assume that one or more migrations have **already been applied**. This prevents generated migrations from failing when they try to create tables and fields that already exist.
+
+When you add Prisma Migrate to an existing project, your initial migration contains all the SQL required to recreate the state of the database **before you started using Prisma Migrate**:
+
+![The image shows a database labelled 'Existing database', and a list of existing database features next to it - 24 tables, 13 relationships, 92 fields, 3 indexes. An arrow labelled 'represented by' connects the database features list to a box that represents a migration. The existing databases's features are represented by a single migration.](existing-database.png)
 
 To initialize a new migration history:
 
 1. If you have a `prisma/migrations` folder, delete, move, rename, or archive this folder.
 
-1. Run the following command to create the first migration **without applying it**, in case you need to [modify the initial migration](#work-around-features-not-supported-by-prisma-schema-language):
+1. Run the following command to create a `migrations` directory inside with your preferered name. This example will use `0_init` for the migration name
 
-   ```terminal
-   prisma migrate dev --name initial-migration --create-only
-   ```
+Run the following commands to create the first migration **without applying it**, in case you need to modify the initial migration:
 
-   Prisma Migrate will create a new migration directory with a SQL migration file in it:
+```terminal
+mkdir -p prisma/migrations/0_init
+```
 
+   <Admonition>
+
+Then `0_` is important because Prisma Migrate applies migrations in a [lexographic order](https://en.wikipedia.org/wiki/Lexicographic_order). You can use a different value such as the current timestamp.
+
+   </Admonition>
+
+1. Generate a migration and save it to a file using `prisma migrate diff`
+
+   ```terminal
+   npx prisma migrate diff \
+   --from-empty \
+   --to-schema-datamodel prisma/schema.prisma \
+   --script > prisma/migrations/0_init/migration.sql
    ```
-   ./prisma/migrations/20210426141759_initial-migration
-   ```
+
+1. Review the generated migration
 
 ### Work around features not supported by Prisma Schema Language
```

Comment:
Does this section still make sense in this context?
Do we have this in all the other places where we explain to use baselining?

Reply:
We don't. I'll take another look at it and make any necessary updates.

Reply:
It's fine if we have 1 location that shows the full complexity, and the other places that use the same process skip over some of the details.